scriptmodules/supplementary/runcommand.sh

```diff
@@ -19,7 +19,7 @@ function _update_hook_runcommand() {
 }
 
 function depends_runcommand() {
-    local depends=()
+    local depends=(python-pyudev)
```

Comment:
I may change the startup dependency checks to be a "quick check" without using dpkg.